students/mkdir01/lesson07/html_render.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+
+
+class Element:
+    tags = ["<html>", "</html>"]
```

Comment:
it is simpler to have single `tag` attribute since deriving start/end tags is easier to do programmatically